### DISCUSSION TITLE
Max Zoom in Frigate+

### DISCUSSION THREAD

**Discussion by borgmanJeremy** [blakeblackshear/frigate#11936] [Report a Bug] [answered]
### Describe the problem you are having

When tagging images in Frigate+ the zoom is restricted so the image doesnt "clip" at all. This restricts the maximum zoom allowed. When tagging faces this makes it pretty hard to get tight bounding boxes. I suggest the maximum zoom have no upper limit. 

### Steps to reproduce

Tag images in Frigate+ in the browser (tested in firefox and chromium)

Repeatedly zoom in

Note that the maximum zoom level is capped so no clipping occurs. 

### Version

Frigate+ doesnt seem to show a version

### Frigate config file

```yaml
Not relevant for Frigate+
```

### Relevant log output

```shell
Not Relevant for Frigate+
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Not relevant for Frigate+

**Comment by NickM-27** [comment, marked as answer]
this is not a bug and working as expected. See the frigate+ roadmap https://github.com/orgs/frigate-nvr/projects/2/views/1
